plynx/web/user.py

```diff
@@ -23,6 +25,31 @@ def get_auth_token():
     })
 
 
+@app.route('/plynx/api/v0/register', methods=['POST'])
+@handle_errors
+def post_register():
+    email = request.headers.get('email').lower()
+    username = request.headers.get('username').lower() 
+    password = request.headers.get('password')
+
+    dic = validate_user(
+        email=email, 
+        username=username, 
+        password=password, 
+        confpassword=request.headers.get('confpassword')
+    )
+
+    if dic['success']:
```

Comment:
Please try to avoid
1. dictionaries unless this is really necessary. Just in my experience, dict is not good for stable code: field names get changed. Complicated structures like dict['res'][0]['status'] is super dangerous and ugly (not this case though)
2. outside reader should be able to understand what `validate_user` does and return. Not super clear that it is a dictionary with a field `success`. Normally the function returns True or False or raises an exception. In plynx I believe we were using exceptions